### ISSUE TITLE
How to set up auto-completion on ajax?

### ISSUE THREAD

**Issue by Batisska** [zofe/rapyd-admin#444]
Explain to me please. I have a field city (text). I'm trying to add autocomplete.

MyСontroller
`$edit->autocomplete('city', 'City')->remote ('null', "name", "get-city");`

Route
`Route::get ('/ get-city', 'AjaxController@getCity');`

AjaxСontroller
`retourn response () -> json (['test1', 'test2']);`

I get an error
count (): Parameter must be an array or an object that implements countable.

How to set up auto-completion on ajax?

**Comment by zofe** [comment]
The result should a collection, where there are property you declare in
your field
so basically something like
´´´[{"id":1,"city":"555", "name":"London"}, {"id":2,"city":"666",
"name":"Rome"}]´´´

 the json output of a eloquent collection for example:
´´´return App\City::where('name', 'like',
\Input::get("q")."%")->take('10')->get();´´´

Il giorno mer 31 ott 2018 alle ore 13:10 Андрианов Сергей <
notifications@github.com> ha scritto:

**Comment by Batisska** [comment]
Yes, I forgot to add that requests do not go away after changing the field.
